csharp/src/Ice/Internal/IPEndpointI.cs

```diff
@@ -117,7 +120,7 @@ public override bool equivalent(EndpointI endpoint)
         }
         IPEndpointI ipEndpointI = (IPEndpointI)endpoint;
         return ipEndpointI.type() == type() &&
-            ipEndpointI.host_.Equals(host_, StringComparison.Ordinal) &&
+            ipEndpointI._normalizedHost.Equals(_normalizedHost, StringComparison.Ordinal) &&
```

Comment:
I would use operator== to compare these strings.

See https://learn.microsoft.com/en-us/dotnet/csharp/how-to/compare-strings.

Reply:
> Make the intent of your code clear by calling an overload that explicitly specifies the type of comparison to perform.

Isn't that what we do here? Afaik there is a linter rule for this.

Reply:
See for example https://github.com/icerpc/icerpc-csharp/blob/fc251d6252e0135d9312dbac6b46e3dae3b23b49/src/IceRpc/ServerAddress.cs#L179

In general, we should use `==` for string comparison in C#, not Equals.